gadget/install/install_dummy.go

```diff
@@ -31,6 +31,12 @@ import (
 	"github.com/snapcore/snapd/timings"
 )
 
+var SysfsPathForBlockDevice = sysfsPathForBlockDevice
```

Comment:
is intentional to also export this or just to mock it? otherwise this could be done in export_test.go

Reply:
I had to add this one so `nosecboot` tests would know the definition of `SysfsPathForBlockDevice` - if I move the definition from install{_dummy}.go to export_test.go then it won't compile, or am I missing something here?

Reply:
this still feels wrong, I'll play locally to untangle this

Reply:
I pushed some tweaks to avoid this, please take a look

Reply:
Looks fine, thanks